addons/l10n_es_edi_facturae/models/account_tax.py

```diff
@@ -0,0 +1,73 @@
+from odoo import _, fields, models
+
+
+class AccountTax(models.Model):
+    _inherit = 'account.tax'
+    l10n_es_edi_facturae_tax_type = fields.Selection([
+        ('01', _('Value-Added Tax')),
```

Comment:
no need to translate, the orm already does it

Reply:
@JulienAlardot I suppose you are going to change it then?